### ISSUE TITLE
Winter 2020-2021 corpora publication thread

### ISSUE THREAD

**Issue by ctschroeder** [CopticScriptorium/corpora#67]
- [ ] AP in Winter [2020-21 thread](https://github.com/CopticScriptorium/ap-dev/issues/89)
    - [ ] AP 37-39 (probably next time)
    - [x] AP 43-45

- [x] Apa Johannes canons see [Johannes issue 12](https://github.com/CopticScriptorium/johannes-canons-dev/issues/12)

- [x] Marcion Corpora (test versions now in ANNIS!) [see thread for status details](https://github.com/CopticScriptorium/budge-dev/issues/9)
  - [x] Mysteries of John 
  - [x] Life of John the Monk
  - [x] Timothy's Discourse on Abbaton
  - [x] Asketikon of Ephrem

- [x] NT corpus with entity linking (test version now in ANNIS!)
  - [x] Finish ambiguous names spreadsheet
  - [x] Re-run NLP and test

- [x] Magical Papyri (4 documents)

**Comment by ctschroeder** [comment]
@amir-zeldes can you please add Marcion corpora here? Thanks so much!!!

**Comment by amir-zeldes** [comment]
I don't have a closed list yet, but I can add some ones we already have in the pipeline

**Comment by ctschroeder** [comment]
Thanks so much! Can you also mark everything from the last release as "published' in Gitdox @amir-zeldes?

**Comment by amir-zeldes** [comment]
Done

**Comment by ctschroeder** [comment]
@tsiuda should be done with the translation spans of her doc in the Life of John and also her doc in the Mysteries of John by mid March, so in time for the release.  Will update the other files and threads today or this weekend.

**Comment by tsiuda** [comment]
Life of John translation spans completed today (5 March 2021); Mysteries is now in process.

**Comment by ctschroeder** [comment]
Life of John the monk metadata: author, PATHS link, Greek edition need investigating

**Comment by ctschroeder** [comment]
Pseudo ephrem corpus should be pseudo.ephrem. Document titles can remain psephrem.asketikon.01 etc.  (Back when we published the letter of Pseudo Ephrem we decided to make a pseudo.ephrem corpus in which we would put all the pseudo ephrem material.)

**Comment by amir-zeldes** [comment]
> Pseudo ephrem corpus should be pseudo.ephrem. Document titles can remain psephrem.asketikon.01 etc. (Back when we published the letter of Pseudo Ephrem we decided to make a pseudo.ephrem corpus in which we would put all the pseudo ephrem material.)

Wait, are you saying the asketikon documents should be merged into the earlier pseudo ephrem corpus? Or should they stay two separate corpora?

**Comment by ctschroeder** [comment]
Put them in the existing pseudo.ephrem corpus. We had a huge discussion about this back when we published the first pseudo.ephrem letter and decided to have a pseudo.ephrem corpus rather than a bunch of smaller ps-ephrem corpora with one or two documents each.

**Comment by amir-zeldes** [comment]
OK, corpus renamed for both documents - @lancealanmartin asketikon can now be edited normally

**Comment by amir-zeldes** [comment]
I am setting the version_n to 4.1.0 with version_date of 2021-03-31

@lancealanmartin can you fill in entities quality metadata? I'm not sure which ones are automatic vs. checked (if checking is very minimal outside named entities please use automatic)

**Comment by lancealanmartin** [comment]
I will fill them in soon.

**Comment by lancealanmartin** [comment]
The metadata for all Marcion docs except john.mysteries.02 is complete.

@ctschroeder can you check the cts_urn field when you get a chance?

**Comment by ctschroeder** [comment]
Brad’s home for a long weekend so I should get to this early next week.

**Comment by ctschroeder** [comment]
I unchecked the items above I am still working on them. Thanks!

**Comment by lancealanmartin** [comment]
The Kyprianos metadata is almost finished. I have one question:

Ed mentioned that each of the Kyprianos papyri has both a manuscript number (KYP M71) and a text number (KYP T53). The doc names for one of the papyri use the manuscript number and the others use the text number. Should we streamline these so that the titles all refer to either the manuscript or the text?

**Comment by amir-zeldes** [comment]
Good point - I think if there's a chance that one MS can contain two texts, then it would make sense to use the text numbers, since each text would receive a different doc (cf. AP)

**Comment by lancealanmartin** [comment]
Great! I changed KYP M71 to KYP T53.

**Comment by ctschroeder** [comment]
We need to talk for a minute about the magical papyri metadata. See this thread https://github.com/CopticScriptorium/papyri.info-dev/issues/4

**Comment by ctschroeder** [comment]
Johannes, Mysteries, and Life are all ready. The blocker on the AP is me. Christie has been done for a while. I'll get to the other Budge and the papyri and the AP next.

**Comment by ctschroeder** [comment]
papyri ready

**Comment by ctschroeder** [comment]
Abbaton ready
Asceticon ready except for Greek_ed

**Comment by amir-zeldes** [comment]
OK, everything is all set except:

  * AP
  * Asketikon (is meta now complete?)
  * Johannes - the question about a page with 1 column

Rest incl. revised NT and papyri is up in ANNIS and GH corpora dev, will only pull to master and CTS repo once the last points are resolved.

**Comment by lancealanmartin** [comment]
I added the Greek edition to Asketikon. It should be ready.

**Comment by amir-zeldes** [comment]
I can add entities, no worries

**Comment by amir-zeldes** [comment]
OK, all done - just waiting on the answer to the 1-column page of Johannes, then it's a wrap!

**Comment by amir-zeldes** [comment]
We're done! I'm not bundling a release tag on GH yet in case we need to do any last minute hotfixes, but ANNIS, GH master and CTS repo should all be up to date now. Let me know if you notice any issues and I can add the tag in a couple of days.
